<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SizeDialog</class>
 <widget class="QDialog" name="SizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Size</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="widthLayout">
     <item>
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="widthSlider">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="widthDisplay">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="heightLayout">
     <item>
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="heightSlider">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="heightDisplay">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="oKButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>oKButton</sender>
   <signal>clicked()</signal>
   <receiver>SizeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SizeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widthDisplay</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>heightDisplay</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
